package.json

```diff
@@ -26,44 +26,45 @@
         "changelog": "git-cliff -o CHANGELOG.md"
     },
     "devDependencies": {
-        "@eslint/js": "^9.28.0",
-        "@playwright/test": "^1.52.0",
+        "@eslint/js": "^9.29.0",
+        "@playwright/test": "^1.53.1",
         "@solidjs/testing-library": "^0.8.10",
         "@testing-library/user-event": "^14.6.1",
         "@trivago/prettier-plugin-sort-imports": "^5.2.2",
         "@types/node": "^22.15.29",
         "@webbtc/webln-types": "^3.0.0",
-        "axios": "^1.9.0",
-        "eslint": "^9.28.0",
+        "axios": "^1.10.0",
+        "eslint": "^9.29.0",
         "eslint-plugin-solid": "^0.14.5",
         "git-cliff": "^2.9.1",
         "globals": "^16.2.0",
-        "prettier": "^3.5.3",
-        "sass-embedded": "^1.88.0",
+        "prettier": "^3.6.1",
+        "sass-embedded": "^1.89.2",
         "typescript": "^5.8.3",
-        "typescript-eslint": "^8.34.0",
+        "typescript-eslint": "^8.35.0",
         "vite": "^6.3.5",
         "vite-plugin-mkcert": "^1.17.8",
         "vite-plugin-node-polyfills": "^0.23.0",
-        "vite-plugin-solid": "^2.11.6",
+        "vite-plugin-solid": "^2.11.7",
         "vite-plugin-top-level-await": "^1.5.0",
-        "vite-plugin-wasm": "^3.4.1"
+        "vite-plugin-wasm": "^3.4.1",
+        "vitest": "^3.2.4"
     },
     "dependencies": {
         "@bitcoinerlab/secp256k1": "^1.2.0",
         "@fontsource/noto-mono": "^5.2.5",
         "@fontsource/noto-sans": "^5.2.7",
-        "@ledgerhq/hw-app-eth": "^6.45.6",
-        "@ledgerhq/hw-transport-webhid": "^6.30.2",
-        "@reown/appkit": "1.6.9",
-        "@reown/appkit-adapter-ethers": "1.6.9",
+        "@ledgerhq/hw-app-eth": "^6.38.0",
+        "@ledgerhq/hw-transport-webhid": "^6.30.3",
```

Comment:
That's downgrading the hw-app-eth? And needs testing with Ledger Nano X and Nano S (which does not support EIP-712)